jdaviz/configs/default/plugins/export/export.vue

```diff
@@ -89,7 +89,7 @@
       </plugin-inline-select>
     </div>
 
-    <div v-if="subset_items.length > 0">
+    <div v-if="subset_items.length > 0 & config !='specviz'">

```

Comment:
is this because this only supports spatial subsets?  If so - we should add that as a filter to the subset component so that they are also excluded in cubeviz and then the length check should automatically work to hide the section

Reply:
done